src/server/game/Scripting/ScriptMgr.cpp

```diff
@@ -2431,7 +2431,8 @@ void ScriptMgr::OnAreaTriggerEntityUpdate(AreaTrigger* areaTrigger, uint32 diff)
     GET_SCRIPT(AreaTriggerEntityScript, areaTrigger->GetScriptId(), tmpscript);
     tmpscript->OnUpdate(areaTrigger, diff);
 }
-
+// place include here to signify its debug intention only!!!
+#include "AreaTriggerTemplate.h"
```

Comment:
can you explain it?

i'm going to change
```
player->SummonCreature(WORLD_TRIGGER, areaTrigger->GetPosition(), TEMPSUMMON_TIMED_DESPAWN, areaTrigger->GetMiscTemplate()->TimeToTarget);
```
to get the TimeToTarget directly from areatrigger, because it can change during runtime
can i remove the include then?